source/_patterns/01-elements/_form-elements.scss

```diff
@@ -134,3 +134,16 @@
 		padding: to-rem($pxValue: 6) to-rem($pxValue: 10);
 	}
 }
+
+// enhance the clickable area for checkbox and radio elements
+%form-element-enhanced-clickable-area {
+	&:has(+ label) {
+		&::after {
+			content: "";
+			width: calc(var(--db-form-element-dimensions) + 4px);
```

Comment:
You use `to-rem` everywhere, is it purpose that you use `px` in this case and for `translateX`?

Reply:
no